src/Wpf.Ui.Demo/App.xaml.cs

```diff
@@ -49,17 +49,14 @@ private async void OnStartup(object sender, StartupEventArgs e)
     /// <summary>
     /// Configures the services for the application.
     /// </summary>
-    private void ConfigureServices(HostBuilderContext context, IServiceCollection services)
+    private static void ConfigureServices(HostBuilderContext context, IServiceCollection services)
     {
+        //Add all wpf services
+        services.AddWpfUiServices();
```

Comment:
Why not just use one method that adds all the related wpf UI services?

Reply:
And I suppose I Page Service implementation should be in  wpf ui.

Reply:
> Why not just use one method that adds all the related wpf UI services?

Because someone may not want to use all of them and because not everyone wants to use Dependency Injection

> And I suppose IPageService implementation should be in wpf ui.

No, because this implementation should be independent of the ServiceCollection provider